src/engraving/libmscore/barline.cpp

```diff
@@ -1058,6 +1058,24 @@ void BarLine::startEdit(EditData& ed)
     ed.addData(bed);
 }
 
+bool BarLine::edit(EditData& ed)
+{
+    bool local = ed.control() || segment()->isBarLineType() || spanStaff() != score()->staff(staffIdx())->barLineSpan();
+    if (ed.key == Qt::Key_Up && spanStaff() || ed.key == Qt::Key_Down && !spanStaff()) {
+        if (local) {
+            BarLine* b = toBarLine(segment()->element(staffIdx() * VOICES));
+            if (b) {
+                b->undoChangeProperty(Pid::BARLINE_SPAN, !spanStaff());
+            }
+        } else {
+            score()->staff(staffIdx())->undoChangeProperty(Pid::STAFF_BARLINE_SPAN, !spanStaff());
+        }
+        return true;
+    }
+    return Element::edit(ed);
+}
+
+
```

Comment:
Now this one extra empty lined cause the codestyle check to fail.
But that seems to be the least problem of your latest changes, which don#t seem to build at all?